src/connection/stream.rs

```diff
@@ -207,48 +217,33 @@ impl futures::stream::Stream for Stream {
             return Poll::Ready(None)
         }
 
-        ready!(self.send_pending_window_update(cx))?;
+        ready!(self.send_window_update(cx))?;
```

Comment:
I know it was the case before and it may be unlikely to happen that `sender.poll_ready()` is `Poll::Pending`, but I'm wondering why we should ever block reading (via the `ready!` here) from the already buffered data in `Stream::poll_read` or `Stream::poll_next` on the ability to send a window update to the _bounded_ `StreamCommand` channel beforehand. It seems to potentially just create more opportunity for situations as observed in #104, if `Connection::next_stream` is currently is currently "stuck" in a `send().await`.